### PR TITLE
Update to python-0.9.0 - with ARM and windows

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Comment by jan-janssen** [comment]
After the successful builds in https://github.com/conda-forge/lancedb-feedstock/pull/10 and https://github.com/conda-forge/lancedb-feedstock/pull/11 I am confident that this pull request should pass all the test.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,49 +1,56 @@
...
+  skip: true  # [py<=38]
 
 requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ stdlib("c") }}
+    - {{ compiler("rust") }}
+    - cargo-bundle-licenses
+    - pkg-config
+    - protobuf
```
Comment (xhochy) [inline recipe/meta.yaml:25]: You probably only want `protoc` here.
```suggestion
    - libprotobuf
```

**Comment by jan-janssen** [comment]
@moritzwilksch and @mediocretech Can you take a look at this pull request?

**Review by moritzwilksch — APPROVED** [review]

**Comment by moritzwilksch** [comment]
thanks a lot @jan-janssen!

**Pull request merged**